ksz8863/examples/switch_mode/main/switch_main.c

```diff
@@ -289,6 +292,13 @@ esp_err_t ksz8863_board_specific_init(esp_eth_handle_t eth_handle)
     return ret;
 }
 
+static esp_err_t eth_incoming_data_handler(esp_eth_handle_t eth_handle, uint8_t *buffer, uint32_t length, void *priv)
```

Comment:
I presume this is used only for debug purposes and it will be removed, am I right?